include/tvm/runtime/c_runtime_api.h

```diff
@@ -32,6 +32,11 @@
  *  The common flow is:
  *   - Use TVMFuncListGlobalNames to get global function name
  *   - Use TVMFuncCall to call these functions.
+ *
+ *  Possible return values of the API functions:
```

Comment:
Given the number of categories are small atm, I feel it may not be necessary to introduce an error code enum

Reply:
it's more for organization to document the 0/-1/-2 convention. perhaps you're right...it would be nice to align on an error code type, but it could be a different PR